pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -267,6 +267,18 @@ func (og *operationGenerator) GenerateAttachVolumeFunc(
 			volumeToAttach.VolumeSpec, volumeToAttach.NodeName)
 
 		if attachErr != nil {
+			if derr, ok := attachErr.(*util.DanglingAttachError); ok {
+				addErr := actualStateOfWorld.MarkVolumeAsAttached(
+					v1.UniqueVolumeName(""),
```

Comment:
Is the empty volume name going to cause a problem... can we have a comment as to why not?

Reply:
nah - the if volume spec is specified in `MarkVolumeAsAttached` then unique name is picked from volume spec rather than from first parameter. I am just filling out first parameter to satisfy function contract.